### DISCUSSION TITLE
[Feature Request] 自动跳转至对应语言页

### DISCUSSION THREAD

**Discussion by BrandonStudio** [spencerwooo/onedrive-vercel-index#431] [Ideas]
从
`https://oneindex.address/path`
跳转至
`https://oneindex.address/zh-CN/path`
或
`https://oneindex.address/en-US/path`
前者是中文页，后者是英文页。

**Comment by spencerwooo** [comment]
其实没太懂，现在不是自动跳转吗只是默认 `/en` -> `/`？

**Reply by BrandonStudio** [reply]
具体来说中文浏览器环境下`https://webdrive.brandonstudio.site/Apps/`应跳转为`https://webdrive.brandonstudio.site/zh-CN/Apps/`

**Reply by BrandonStudio** [reply]
就是说在后面有路径的时候不会自动跳成中文的

**Reply by spencerwooo** [reply]
会自动跳啊，前缀是 `/zh-CN` 的情况下，导航到各个文件夹都是中文啊

**Reply by spencerwooo** [reply]
Next.js 的多语言是 1. 根据浏览器语言 2. 找 `NEXT_LOCALE` 这个 Cookie 来判断的

你要找的是不是：https://nextjs.org/docs/advanced-features/i18n-routing#automatic-locale-detection

如果你的意思是：中文浏览器，访问子路径，没有自动 locale detection — 那是 Next.js 只支持在 `/` 检测语言的问题 😄

**Reply by BrandonStudio** [reply]
是前缀没有zh-CN的情况下
我给出的是不含locale的地址，路由时应该自动根据客户端的语言信息确定locale从而自动跳转
就像`https://docs.microsoft.com/dotnet`会根据客户端信息跳转成`https://docs.microsoft.com/zh-CN/dotnet`或`https://docs.microsoft.com/en-US/dotnet`一样

**Reply by spencerwooo** [reply]
那就是「子路径」没有自动语言检测了，是我上面提到的 Next.js 的问题了。
